docs/intro.md

```diff
@@ -49,6 +49,7 @@ Then build the compiler:
 ```
 cargo build
 ```
+Make sure to setup the git hooks via `/bin/sh setup_hooks.sh`.
```

Comment:
This should be formatted as a paragraph and we should provide some explanation about why we're asking people to do this (we enforce a formatting style in the CI so it's much easier if you set this up to make the commit process easier).